src/Applications/LDAS_App/sample_config_files/LADAS/exeinp.txt.centralC180

```diff
@@ -0,0 +1,623 @@
+####################################################################################
```

Comment:
Same comment as for `exeinp.txt.atmensC90` above.
If we don't include the resolution in the sample (or use a resolution placeholder string), then the sample config files can be labeled more simply and more concisely, e.g., `exeinp.txt.Hy4dEnVar.atmens` and `exeinp.txt.Hy4dEnVar.central`.  This clarifies that the samples are not for the 3dVar system.  This info should also be included as a comment at the top of the exeinp.txt sample files. 
Also, we may need to include more info about how the atm ens is used in these examples.  This is the latest config that uses the atm ens to create ensemble forcing for the LDAS associated with the central member, right?

Reply:
these are good file names with clear information on coupling targets. But we need to instruct user on how these files are used  ( as reference to edit the complete expinp file generated by ldas_setup sample --expinp) . 
We can add some info on the connection of ldas to atm ens , particularly ens forcing and ens incr ( same  Nens on both sides) .

Reply:
Ok, thanks.  Please go  ahead and implement these changes.

Reply:
I just pushed updates.